### PR TITLE
[WIP][GUI][DB](#35)List view 

### LINKED ISSUE #35: [GUI][DB] From Database entry to GUI
#### Problem Description

So, now we have all the tools we need (independent components) we can starting building the applications.

Either @cswezey and @chet0322  should work with @JeewakaSankalpa to get database entry to GUI and display them.

### PR DESCRIPTION
**Pull request by JeewakaSankalpa**
- [ ] closes #35 
- [ ] tests added / passed
- [ ] passes ``git diff upstream/master | flake8 --diff``

### CODE REVIEW

**Comment by JeewakaSankalpa** [comment]
Completed this Section.
Markdown file can now be loaded from the DB and Viewed in the Markdown Section and the related HTML version is displayed correctly in the Rightmost Widget. 
I'll Demo it in class on thurseday

File: captslog/db/DBHandler.py [hunk 1]
```diff
@@ -126,6 +126,26 @@ def search_entries_by_created_date(self, date):
                       ' the target machine actively refused it')
                 return True
         return False
+    def search_entries_by_id(self, id):
```
Comment (jaehoonhwang) [inline captslog/db/DBHandler.py]: Write a unit test for this function.

File: captslog/db/DBHandler.py [hunk 2]
```diff
@@ -194,6 +214,22 @@ def delete_entries(self, _id):
                   ' the target machine actively refused it')
             return False
 
+    def get_all(self):
```
Comment (jaehoonhwang) [inline captslog/db/DBHandler.py]: Unit test required.

Comment (JeewakaSankalpa) [inline captslog/db/DBHandler.py]: Done

File: captslog/db/DBHandler.py (hunk 1 above)
Comment (JeewakaSankalpa) [inline captslog/db/DBHandler.py]: Done

**Pull request merged**